apps/web/src/components/workflow/preview/ErrorPrettyRender.tsx

```diff
@@ -1,37 +1,74 @@
 import { css } from '@novu/novui/css';
-import { Text } from '@novu/novui';
-import { Center } from '@novu/novui/jsx';
-import { IconErrorOutline } from '@novu/novui/icons';
+import { Button, Text, Title } from '@novu/novui';
+import { Center, Stack, VStack } from '@novu/novui/jsx';
+import { IconErrorOutline, IconExpandLess, IconExpandMore } from '@novu/novui/icons';
+import { useDisclosure } from '@mantine/hooks';
+import { hstack } from '@novu/novui/patterns';
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** Is it worthwhile to add an `HSack` component into `novui`? I've seen a few places we use it.‏

Reply:
@rifont It is a pre-built component by Panda, but the beauty of it is that it's also a `pattern` that generates a `className`! This allows us to style semantic elements (button in this case) without having to use a polymorphic component (aka adding `as="button"`), which tends to help TypeScript in my experience